experiments/AtmosGCM/heldsuarez.jl

```diff
@@ -173,73 +189,85 @@ function held_suarez_forcing!(
     k_v = k_f * height_factor
 
     # Apply Held-Suarez forcing
-    source.ρu -= k_v * projection_tangential(bl, aux, ρu)
+    source.ρu -= k_v * ρu
```

Comment:
Why are we back to damping vertical velocities too, rather than just horizontal?

Reply:
It looks like they damp all velocity components in Held & Suarez 1994.

Reply:
No, they don't. The model is hydrostatic, hence `v` is horizontal velocity only.

Reply:
Makes sense. I'll revert to original state.